recipe/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
     - fsspec
     - jsonargparse[signatures] >=3.17.0, <=4.9.0
     - lightning-utilities >=0.3.0, !=0.4.0
```

Comment:
> pytorch-lightning 1.8.1 has requirement lightning-utilities==0.3.*, but you have lightning-utilities 0.5.0

Might need some updates on https://github.com/conda-forge/pytorch-lightning-feedstock?

Reply:
Ok, new traceback error:

```python-traceback
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/lightning-flash_1681333698504/test_tmp/run_test.py", line 2, in <module>
    import flash
  File "/home/conda/feedstock_root/build_artifacts/lightning-flash_1681333698504/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.10/site-packages/flash/__init__.py", line 22, in <module>
    from flash.core.data.callback import FlashCallback
  File "/home/conda/feedstock_root/build_artifacts/lightning-flash_1681333698504/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.10/site-packages/flash/core/data/callback.py", line 8, in <module>
    from flash.core.data.utils import _STAGES_PREFIX
  File "/home/conda/feedstock_root/build_artifacts/lightning-flash_1681333698504/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/python3.10/site-packages/flash/core/data/utils.py", line 22, in <module>
    from pytorch_lightning.utilities.apply_func import apply_to_collection
ModuleNotFoundError: No module named 'pytorch_lightning.utilities.apply_func'
Tests failed for lightning-flash-0.8.1.post0-pyhd8ed1ab_0.conda - moving package to /home/conda/feedstock_root/build_artifacts/broken
WARNING:conda_build.build:Tests failed for lightning-flash-0.8.1.post0-pyhd8ed1ab_0.conda - moving package to /home/conda/feedstock_root/build_artifacts/broken
TESTS FAILED: lightning-flash-0.8.1.post0-pyhd8ed1ab_0.conda
```

May need to pin to `pytorch-lightning<2`?